packages/itwinui-css/src/tile/tile.scss

```
@@ -150,9 +154,12 @@ $thumbnail-selector: ':is(.iui-thumbnail-icon, .iui-tile-thumbnail-picture)';
 }
 
 @mixin iui-folder {
-  display: inline-flex;
-  flex-direction: row;
-  flex-wrap: nowrap;
+  display: grid;
+  grid-template-areas:
+    'thumbnail name'
+    'thumbnail content';
+  grid-template-columns: 3fr 7fr;
+  grid-template-rows: 1.5fr 2.5fr;
```

Since @FlyersPh9 is out, here's my next best suggestion for the grid:
```suggestion
  grid-template-columns: 1fr 2fr;
  grid-template-rows: auto 1fr;
```

the columns preserve the 1:2 ratio from before, while the rows will prefer to keep the "name" part fixed and let the "content" part expand.

you will likely need to add some gap between the name and content. I think there was a `margin-bottom` before which you removed. Reintroducing that might fix the visuals.

---

This is working now, as in no more visual issues. Thanks for the suggestion